lib/streamlit/DeltaGenerator.py

```diff
@@ -2095,6 +2096,63 @@ def file_uploader(
         file_datas = [get_encoded_file_data(file.data, encoding) for file in files]
         return file_datas if accept_multiple_files else file_datas[0]
 
+    @_with_element
+    def color_picker(self, element, label, value=None, key=None):
+        """Display a color picker widget.
+
+        Parameters
+        ----------
+        label : str
+            A short label explaining to the user what this input is for.
+        value : any
```

Comment:
Depending on the API decision I commented on in the python tests, the type and description here need to be updated